[2024-09-01 19:13:15,029] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: exchange_rates_dag.run_main_script scheduled__2024-06-02T00:00:00+00:00 [queued]>
[2024-09-01 19:13:15,185] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: exchange_rates_dag.run_main_script scheduled__2024-06-02T00:00:00+00:00 [queued]>
[2024-09-01 19:13:15,197] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-09-01 19:13:15,202] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-09-01 19:13:15,206] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-09-01 19:13:15,414] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): run_main_script> on 2024-06-02 00:00:00+00:00
[2024-09-01 19:13:15,464] {standard_task_runner.py:52} INFO - Started process 1276 to run task
[2024-09-01 19:13:15,516] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'exchange_rates_dag', 'run_main_script', 'scheduled__2024-06-02T00:00:00+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates_dag.py', '--cfg-path', '/tmp/tmp8nptx_o5', '--error-file', '/tmp/tmpfuttwh7d']
[2024-09-01 19:13:15,542] {standard_task_runner.py:80} INFO - Job 158: Subtask run_main_script
[2024-09-01 19:13:16,575] {task_command.py:371} INFO - Running <TaskInstance: exchange_rates_dag.run_main_script scheduled__2024-06-02T00:00:00+00:00 [running]> on host 6de2165e2444
[2024-09-01 19:13:17,658] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rates_dag
AIRFLOW_CTX_TASK_ID=run_main_script
AIRFLOW_CTX_EXECUTION_DATE=2024-06-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-02T00:00:00+00:00
[2024-09-01 19:13:40,017] {python.py:173} INFO - Done. Returned value was: None
[2024-09-01 19:13:41,091] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=exchange_rates_dag, task_id=run_main_script, execution_date=20240602T000000, start_date=20240901T191315, end_date=20240901T191341
[2024-09-01 19:13:41,309] {local_task_job.py:84} ERROR - Received SIGTERM. Terminating subprocesses
[2024-09-01 19:13:41,463] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 1276. PIDs of all processes in the group: [1276]
[2024-09-01 19:13:41,563] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 1276
[2024-09-01 19:13:41,656] {taskinstance.py:1561} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-09-01 19:13:41,716] {standard_task_runner.py:97} ERROR - Failed to execute job 158 for task run_main_script (Task received SIGTERM signal; 1276)
[2024-09-01 19:13:42,008] {process_utils.py:75} INFO - Process psutil.Process(pid=1276, status='terminated', exitcode=1, started='19:13:14') (1276) terminated with exit code 1
[2024-09-01 19:13:42,012] {local_task_job.py:156} INFO - Task exited with return code 143
[2024-09-01 19:13:42,305] {taskinstance.py:1420} INFO - Marking task as UP_FOR_RETRY. dag_id=exchange_rates_dag, task_id=run_main_script, execution_date=20240602T000000, start_date=20240901T191315, end_date=20240901T191342
[2024-09-01 19:13:43,086] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
